src/mame/drivers/jubilee.cpp

```diff
@@ -418,19 +418,19 @@ WRITE8_MEMBER(jubilee_state::unk_w)
 	{
 		if (muxlamps == 1)
 			{
-				m_lamp[0] = BIT(data, 0);  /* lamp */
+				m_lamps[0] = BIT(data, 0);  /* lamp */
```

Comment:
Eugh, surely there's a better way than three copies of the same code here?  Maybe `(muxlamps - 1) * 3` or something, and similar for the other repeating pieces of code?